posts/2025-11-04-why-make-a-major-release.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Why we made a major release for Jupyter Book 2 instead of creating a new package"
+date: 2025-11-04
+license: CC-BY-4.0
+authors:
+  - Jupyter Book Team
+---
+
+Last night, we [released Jupyter Book 2](https://github.com/jupyter-book/jupyter-book/releases/tag/v2.0.0).
+This is a major release that introduces the [MyST Document Engine](https://mystmd.org) as the back-end infrastructure that powers Jupyter Book.
+The alpha and beta versions of this release have been in progress since November 2024, and it represents a huge amount of work from the team and community that we are incredibly excited about.
+
+Unfortunately, it also means that many Jupyter Books out there are now broken.
+In particular, users who haven't pinned their Jupyter Book versions to the `1.x` series will likely experience new build failures.
+
+If you are not ready to try out the new version, you can reinstall a version of Jupyter Book compatible with the 1.x series by using commands like:
+
+```shell
+$ pip install "jupyter-book<2"
+OR
+$ pip install "jupyter-book~=1"
+```
+
+
+## Why we didn't create an entirely new package
+
+We considered creating a new Python package to reflect the big switch to a new documentation engine - e.g., `pip install jupyter-book2`.
+This has been done before in the Python system, and is a way to strictly separate the two projects.
+
+However, we see the engine upgrade as a continuation of the project, and not a complete replacement that warrants a new package name. We believe that it provides features that our users will appreciate, and want to make it possible for them to cross the gap between the two versions without much effort.
+
+We do not plan on maintaining the old Sphinx-based engine, and think that releasing the new engine under the same package name signals that intent most clearly.
+
+Finally, we think that having two user-facing products both called "Jupyter Book" creates some confusion: community members and users can end up talking across each other without realising that there are two engines that exist in parallel.
+
+As it goes in open source projects, we discussed the various transition pathways, in which some team members expressed reservations with the chosen path—being concerned about the burden the transition imposed. 
```

Comment:
side-bar: I love the fact that we're explicitly saying that some folks disagree with this decision. IMO that is healthy (and appreciated from my perspective)